pkg/tgwriter/tgwriter.go

```diff
@@ -12,7 +13,7 @@ func New(w http.ResponseWriter, log *zap.SugaredLogger) *TgWriter {
 	w.Header().Set("Content-Type", "application/json")
 
 	return &TgWriter{
-		output: w,
+		output: ioutils.NewWriteFlusher(w),
```

Comment:
This is already done at a few other places. If we are going to wrap the `TgWriter` with a `Flusher` by default, we should remove the occurrence at the rest of the places in the codebase.

Reply:
Was this fixed already? I haven't seen the changes in the code, such as in https://github.com/ipfs/testground/blob/489c147b07706349b2846a5b11824b6b1c82a96a/pkg/server/run.go#L41